### PR TITLE
Include Pluto notebooks

### PR DESCRIPTION
**Pull request by uncomfyhalomacro**
# What I did

I removed the existing sir model and created a Pluto notebook of it in the folder `pluto_nb`. Also included `seir_model` as a new addition.

### CODE REVIEW

**Comment by uncomfyhalomacro** [comment]
Also, we should skip the CI tests for pluto notebooks

**Comment by uncomfyhalomacro** [comment]
well that was inconsistent. nightly just failed here hmm

File: Project.toml [hunk 1]
```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Pluto = "c3e4b0f8-55cb-11ea-2926-15256bba5781"
```
Comment (Azzaare) [inline Project.toml:8]: Is there a need for Pluto to be added as a dependency? The notebooks are independent.

**Review by Azzaare — CHANGES_REQUESTED** [review]
Beside the Pluto entry in the Project.toml, the rest seems good.

I guess, either dismiss my request if the entry is needed (but in that case, please also add the compat entry), or just remove Pluto from the toml file ;)

**Comment by Azzaare** [comment]
> well that was inconsistent. nightly just failed here hmm

You can ignore nightly most of the time. Of course, it is worth it to check the reason it is failing, but something the failure is temporary due to the last build of nightly.
Furthermore, if the failure is really something that should not fail, we should report it to as an issue in the Julia-lang repo.

(I often don't report failure on nightly because of ~~laziness~~ lack of time, so do not take my words too seriously!)

File: Project.toml (hunk 1 above)
Comment (uncomfyhalomacro) [reply Project.toml]: Seems it is not needed since Pluto is portable

**Comment by uncomfyhalomacro** [comment]
> 
> 
> Beside the Pluto entry in the Project.toml, the rest seems good.
> 
> I guess, either dismiss my request if the entry is needed (but in that case, please also add the compat entry), or just remove Pluto from the toml file ;)

I removed it since it is not needed yeah

**Comment by uncomfyhalomacro** [comment]
I guess all is good now

**Review by Panquesito7 — COMMENTED** [review]
I suggest changing the directory name to `pluto_notebooks` as not many people will know what `nb` is.

**Comment by uncomfyhalomacro** [comment]
Okay pls do review. Thanks.

**Review by Panquesito7 — APPROVED** [review]

**Review by Azzaare — APPROVED** [review]

**Pull request merged**